app/controllers/application_controller.rb

```diff
@@ -1,3 +1,9 @@
 class ApplicationController < ActionController::Base
 	
+	def authenticate_admin_user
```

Comment:
indentation
You can use rubocop for fixing it

Reply:
Done.
Tomorrow I'll start using RubyMine with rubocop